main/index.ts

```diff
@@ -46,4 +46,13 @@ export {
 } from '../ipc/schema/packageJson';
 
 export { type OverwriteOptions } from '../ipc/serialPort';
-export { type OpenAppOptions } from '../ipc/openWindow';
+export type {
+    OpenAppOptions,
+    OpenAppOptionsDevice,
+    OpenAppOptionsDeviceSN,
+    OpenAppOptionsDevicePort,
```

Comment:
Only `OpenAppOptions` needs to be exported, the three later ones are not really used by the launcher.